docs/astria.md

```diff
@@ -0,0 +1,145 @@
+
+This document outlines what each of the components do, and provides some basic instructions for running the dev cluster yourself.
+
+## Architecture
+If you have seen any of [Josh's](https://twitter.com/Jskybowen) recent presentations about Astria, you will likely have seen this image:
+![Astria at a high level](images/astria-high-level.png)
+This shows the basic structure of what a shared sequencer is but purposely doesn't dive into any of the details of Astria to keep things simple. However, Astria's general architecture does break down into the basic items shown above.
+
+When expanded to show more detail, Astria's architecture looks like this:
+![Astria Architecture](images/astria-architecture.png)
+Given the diagram above, we can take a look at each of the components that make up Astria in more detail.
+
+### Users
+One of the goals of Astria is to improve the current user experience by having fast finality, Cross Rollup Composability, decentralization, and  sovereignty all in one place and out of the box. There are no hard requirements for rollups to change how their current users interact with what they offer, but additional RPCs would need to be implemented by the rollup in order for their users to take advantage of some of those features.
+
+### Rollups
+Astria is designed for permissionless rollup integration and takes advantage of lazy shared sequencing. This means that rollup developers have total sovereignty over their own execution state and can swap out sequencing layers without fear of being locked in. See the [Introduction to Astria](https://blog.astria.org/introducing-astria/) blog post for more details.
+
+When developing a rollup that utilizes Astria, there are only two interfaces that you need to satisfy:
+1. Rollup <-> Composer: 
+    - For transaction submissions.
+2. Rollup <-> Conductor: 
+    - For receiving data from the sequencing network.
+
+The [Composer](#composer) and [Conductor](#conductor) are explained in more detail in future sections.
+
+![Astria Rollup](images/astria-rollups.png)
+These interfaces utilize [gRPC](https://grpc.io/) to allow for rollup developers to use a wide range of languages for their application. Check out the [astria-protos](https://github.com/astriaorg/astria/tree/main/crates/astria-proto) repo for specific implementation details. This ultimately results in tremendous freedom for developers, giving them control over their own execution and state without needing to worry about the underlying sequencer and data availability infrastructure.
```

Comment:
I think we can (and should) be barebones and not fluffy here. 

Suggested replacement:

Both composer and conductor expose [gRPC](https://grpc.io/) interfaces. See the [astria-protos](https://github.com/astriaorg/astria/tree/main/crates/astria-proto) repo for specific implementation details.